tests/test_expval.py

```diff
@@ -16,89 +16,109 @@
 @pytest.mark.parametrize("theta, phi", list(zip(THETA, PHI)))
 @pytest.mark.parametrize("analytic", [True, False])
 @pytest.mark.parametrize("shots", [8192])
+@pytest.mark.usefixtures("skip_unitary")
 class TestExpval:
     """Test expectation values"""
 
     def test_identity_expectation(self, theta, phi, device, shots, tol):
         """Test that identity expectation value (i.e. the trace) is 1"""
         dev = device(2)
-        dev.apply("RX", wires=[0], par=[theta])
-        dev.apply("RX", wires=[1], par=[phi])
-        dev.apply("CNOT", wires=[0, 1], par=[])
 
-        O = qml.Identity
-        name = "Identity"
+        O1 = qml.Identity(wires=[0])
+        O2 = qml.Identity(wires=[1])
```

Comment:
Curious about this? Are ``01``, ``02`` interpreted as strings?

Reply:
Previously the name of the observable had to be passed to `Device.expval`. With `QubitDevice.expval`, we are passing specific `Observable` instances (e.g. `Identity`). Here `O1` and `O2` are just convenience objects to organize the observables to the beginning of the test case, but they are instances of the `Identity` class.